Kros.KORM/src/Kros.KORM/Query/IDbSet.cs

```diff
@@ -91,12 +91,29 @@ public interface IDbSet<T> : IEnumerable<T>
         /// </example>
         void BulkInsert();
 
+        /// <summary>
+        /// Asynchronously executes bulk insert over pending added items.
```

Comment:
I believe "Asynchronously bulk inserts pending added items." would be better. What is your opinion?